<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigurationWindow</class>
 <widget class="QMainWindow" name="ConfigurationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="okPushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Ok</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelPushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <widget class="QWidget" name="modulesTab">
       <attribute name="title">
        <string>Modules</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>30</y>
          <width>339</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>From this window you can enable/disable modules in the single robots. </string>
        </property>
       </widget>
       <widget class="QCheckBox" name="navigationCheckBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>64</y>
          <width>273</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Exploration + Path planning + Coordination modules</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="semanticCheckBox">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>87</y>
          <width>205</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Semantic information gatherer module</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="victimDetectionCheckBox">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>110</y>
          <width>136</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Victim Detection Module</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="hsvConfigurationTab">
       <attribute name="title">
        <string>HSV Configuration</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Parameters for the HSV Image thresholding used in the Victim detection module </string>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>of every single robot.</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>H MIN</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="hMinSpinBox">
              <property name="minimum">
               <double>0.000000000000000</double>
              </property>
              <property name="maximum">
               <double>255.000000000000000</double>
              </property>
              <property name="value">
               <double>5.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>S MIN</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="sMinSpinBox">
              <property name="minimum">
               <double>0.000000000000000</double>
              </property>
              <property name="maximum">
               <double>255.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>1.000000000000000</double>
              </property>
              <property name="value">
               <double>100.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>V MIN</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="vMinSpinBox">
              <property name="minimum">
               <double>0.000000000000000</double>
              </property>
              <property name="maximum">
               <double>255.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>H MAX</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="hMaxSpinBox">
              <property name="minimum">
               <double>0.000000000000000</double>
              </property>
              <property name="maximum">
               <double>255.000000000000000</double>
              </property>
              <property name="value">
               <double>30.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>S MAX</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="sMaxSpinBox">
              <property name="minimum">
               <double>0.000000000000000</double>
              </property>
              <property name="maximum">
               <double>255.000000000000000</double>
              </property>
              <property name="value">
               <double>230.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>V MAX</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="vMaxSpinBox">
              <property name="minimum">
               <double>0.000000000000000</double>
              </property>
              <property name="maximum">
               <double>255.000000000000000</double>
              </property>
              <property name="value">
               <double>255.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="messageSettingsTab">
       <attribute name="title">
        <string>Message settings</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="2" column="2">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Fault detection messages</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QSlider" name="feedbackMessageSlider">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Feedback Messages</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QSlider" name="semanticMappingSlider">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="4" column="2" colspan="2">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Semantic Mapping messages messages</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QSlider" name="victimDetectionSlider">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Relevance of the message type. </string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Victim detection messages</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QSlider" name="faultDetectionSlider">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="6" column="0" colspan="2">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Minimal priority to show a message:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QSlider" name="minimalPrioritySlider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="6" column="3">
         <widget class="QSpinBox" name="minimalPrioritySpinBox"/>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="victimDetectionSpinBox"/>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="faultSpinBox"/>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="feedbackSpinBox"/>
        </item>
        <item row="4" column="1">
         <widget class="QSpinBox" name="semanticMappingSpinBox"/>
        </item>
        <item row="9" column="0">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="8" column="0">
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="7" column="0">
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
       <zorder>label_12</zorder>
       <zorder>feedbackMessageSlider</zorder>
       <zorder>label_13</zorder>
       <zorder>semanticMappingSlider</zorder>
       <zorder>label_14</zorder>
       <zorder>victimDetectionSlider</zorder>
       <zorder>label_10</zorder>
       <zorder>label_11</zorder>
       <zorder>faultDetectionSlider</zorder>
       <zorder>verticalSpacer_2</zorder>
       <zorder>label_15</zorder>
       <zorder>minimalPrioritySlider</zorder>
       <zorder>minimalPrioritySpinBox</zorder>
       <zorder>victimDetectionSpinBox</zorder>
       <zorder>faultSpinBox</zorder>
       <zorder>feedbackSpinBox</zorder>
       <zorder>semanticMappingSpinBox</zorder>
       <zorder>verticalSpacer_3</zorder>
       <zorder>verticalSpacer_4</zorder>
       <zorder>verticalSpacer_5</zorder>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
